templates/wishlist.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<Container version="2">
+  <Name>Wishlist</Name>
+  <Repository>wingysam/christmas-community</Repository>
+  <Registry>https://hub.docker.com/r/wingysam/christmas-community/</Registry>
+  <Network>bridge</Network>
+  <Shell>sh</Shell>
```

Comment:
```suggestion
```

Only include if you know for a fact the container uses sh (instead of bash) internally.